src/Accelerators/NNPA/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(DLC_SRC_ROOT "${CMAKE_CURRENT_SOURCE_DIR}")
+set(DLC_BIN_ROOT "${CMAKE_CURRENT_BINARY_DIR}")
+
+set(DLC_LIBRARY_PATH ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+set(DLC_RUNTIME_PATH ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+
+set(DLC_ONNX_MLIR_SRC_ROOT ${ONNX_MLIR_SRC_ROOT})
+set(DLC_ONNX_MLIR_BIN_ROOT ${ONNX_MLIR_BIN_ROOT})
+
+# The path to find the zDNN library.
+if(DEFINED ENV{ZDNN_LIBRARY_DIR} AND EXISTS $ENV{ZDNN_LIBRARY_DIR})
```

Comment:
This is looking for the library dir in the environment. In general, cmake files try to avoid using the environment as much as possible. Can you use something like find_package to find the library instead?

Reply:
the zdnn library doesn't have any cmake files in it, so I don't believe I can